docs/migrations/support-slashed-denoms.md

```diff
@@ -0,0 +1,97 @@
+# Migrating from Unsupported Base Denom Slashes to Supporting Slashed BaseDenoms
+
+This document is intended to highlight significant changes which may require more information than presented in the CHANGELOG.
+Any changes that must be done by a user of ibc-go should be documented here.
+
+There are four sections based on the four potential user groups of this document:
+- Chains
+- IBC Apps
+- Relayers
+- IBC Light Clients
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated to bump the version number on major releases.
+```go
+github.com/cosmos/ibc-go/v3 -> github.com/cosmos/ibc-go/v4
```

Comment:
Maybe we should remove this part to avoid confusion...

Reply:
Maybe a asterisk note with special case for this release. I think this is part of the migrations template right?